underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -149,29 +150,48 @@ public static S3AUnderFileSystem createInstance(AlluxioURI uri) {
     }
 
     AmazonS3Client amazonS3Client = new AmazonS3Client(credentials, clientConf);
+    // Set a custom endpoint.
     if (Configuration.containsKey(PropertyKey.UNDERFS_S3_ENDPOINT)) {
       amazonS3Client.setEndpoint(Configuration.get(PropertyKey.UNDERFS_S3_ENDPOINT));
     }
+    if (Configuration.containsKey(PropertyKey.UNDERFS_S3_DISABLE_DNS_BUCKETS)) {
+      // Disable DNS style buckets, this enables path style primary useful for
+      // s3 compliant endpoints.
+      if (Configuration.get(PropertyKey.UNDERFS_S3_DISABLE_DNS_BUCKETS).equals("true")) {
+        S3ClientOptions clientOptions = new S3ClientOptions();
+        clientOptions.withPathStyleAccess(true);
+        amazonS3Client.setS3ClientOptions(clientOptions);
+      }
+    }
+
     TransferManager transferManager = new TransferManager(amazonS3Client);
 
     TransferManagerConfiguration transferConf = new TransferManagerConfiguration();
     transferConf.setMultipartCopyThreshold(MULTIPART_COPY_THRESHOLD);
     transferManager.setConfiguration(transferConf);
 
-    String accountOwnerId = amazonS3Client.getS3AccountOwner().getId();
-    // Gets the owner from user-defined static mapping from S3 canonical user id  to Alluxio
-    // user name.
-    String owner = CommonUtils.getValueFromStaticMapping(
-        Configuration.get(PropertyKey.UNDERFS_S3_OWNER_ID_TO_USERNAME_MAPPING),
-        accountOwnerId);
-    // If there is no user-defined mapping, use the display name.
-    if (owner == null) {
-      owner = amazonS3Client.getS3AccountOwner().getDisplayName();
-    }
-    String accountOwner = owner == null ? accountOwnerId : owner;
+     // Default to readable and writable by the user.
+    short bucketMode = (short) 700;
+    String accountOwner = ""; // There is no known account owner by default.
+    // if ACL enabled inherit bucket acl for all the objects.
+    if (Configuration.containsKey(PropertyKey.UNDERFS_S3A_ENABLE_ACL)) {
```

Comment:
Since you have already set a default value for the key, its unnecessary to check the existence.